README.md

```diff
@@ -1,63 +1,200 @@
 # UINavigationController & TableViews
 
-![ElonMusck](http://i.imgur.com/1i4lr5P.jpg?1)  
+![ElonMusk](http://i.imgur.com/1i4lr5P.jpg?1)  
 
 > When something is important enough, you do it even if the odds are not in your favor. ~[Elon Musk](https://en.wikipedia.org/wiki/Elon_Musk)
- 
 
-## Learning Objectives - The student should be able to..
+## Learning Objectives - The student should be able to...
 
```

Comment:
Moving forward, you can just chop off the - The student should be able to all together so it just reads as

## Learning Objectives